libs/perun/components/src/lib/user-ext-sources-list/user-ext-sources-list.component.ts

```diff
@@ -48,6 +48,8 @@ export class UserExtSourcesListComponent implements AfterViewInit, OnChanges {
   loginHeader: string;
   @Input()
   disableRouting: boolean;
+  @Input()
+  disableSelect = false;
   @Output()
```

Comment:
I think this new value is not necessary. You already removed the 'select' column from displayed columns.

Reply:
It's true. I removed this variable.